src/Common/src/Interop/Windows/Winsock/Interop.WinsockAsync.cs

```diff
@@ -184,213 +176,5 @@ internal struct WSAMsg
             internal WSABuffer controlBuffer;
             internal SocketFlags flags;
         }
-        
```

Comment:
According to the guideline it appears that this is not the organization that we're aiming for:

> If structs/constants will be used on their own without an associated DllImport, or if they may be used with multiple DllImports not in the same file, they should be declared in a separate file.

I don't think these structs can be used by themselves.